collectors/auth0/index.js

```diff
@@ -0,0 +1,23 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2019, Alert Logic, Inc
+ * @doc
+ *
+ * Auth0 System logs collector.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+
+const debug = require('debug') ('index');
+
+const Auth0Collector = require('./auth0_collector').Auth0Collector;
+
+exports.handler = function(event, context) {
+    debug('input event: ', event);
+    Auth0Collector.load().then(function(creds) {
+        var auth0c = new Auth0Collector(context, creds, 'auth0');
```

Comment:
no need to pass 3rd parameter here.

Reply:
k, was following okta example.

Reply:
okta example still needs some clean up as well